### ISSUE TITLE
sysname/reverse dns lookups are inconsistent or lacking after a bulk import

### ISSUE THREAD

**Issue by jmbredal** [jmbredal/test-lp2gh#416]
When bulk importing a number of devices through SeedDB, reverse DNS lookups are deferred to ipdevpoll.

Feedback from users, and our own tests reveal that the deferred sysname updates are spotty, at best. Within two minutes after a new device has been added, ipdevpoll should kick off all jobs for that device, simultaneously. DNS names are retrieved by the dnsname plugin, which runs first in the inventory job.

There are two initial problems here:

1. ipdevpoll will  _not_ run any jobs for a device that is unreachable. If pping has already discovered that the new device doesn't respond to ping requests, the inventory job will never start, and the DNS name is never updated (until the box comes back online).

2. The inventory job must complete successfully for changes to be stored. If the device is configured with an SNMP community, but the inventory job's SNMP requests time out (e.g. using the wrong community, ACL blocking SNMP communication, horrible SNMP implementation on the device), the  updated sysname will never be stored.

The dnsname plugin doesn't care whether the device is working or not, just that the DNS server has a PTR record for the device's IP address.

Suggest these changes:

1. Move the dnsname plugin to a job of its own in the default ipdevpoll config. With its low impact, it can run a lot more often than just every 6 hours.
2. Make ipdevpoll try to run jobs even if a device is down, but move the "device down" test to the base plugin implementation, so that plugins can decide for themselves whether they can do useful things even when the device is down.
